evidently/analyzers/data_drift_analyzer.py

```diff
@@ -22,19 +24,55 @@ def dataset_drift_evaluation(p_values, drift_share=0.5):
 PValueWithConfidence = collections.namedtuple("PValueWithConfidence", ["p_value", "confidence"])
 
 
+@dataclass
+class DataDriftAnalyzerResults:
+    utility_columns: DatasetUtilityColumns
+    cat_feature_names: List[str]
+    num_feature_names: List[str]
+    target_names: List[str]
+    options: DataDriftOptions
+    metrics: dict = field(default_factory=dict)
```

Comment:
I think, `metrics` better to convert to object (probably with some redesigning in structure).

Reply:
done, for now just for metrics in data drift analyzer